src/App.jsx

```diff
@@ -1,35 +1,37 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useEffect, useState } from "react";
+import "./App.css";
+import Home from "./components/Home";
+import Navbar from "./components/Navbar";
+import Header from "./components/Header";
+import { Routes, Route } from "react-router-dom";
+import ArticlesList from "./components/ArticlesList";
+import { getArticles } from "./utils/getFunctions";
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [allArticles, setAllArticles] = useState([]);
+  const [loading, setLoading] = useState(true)
 
+  useEffect(() => {
+    getArticles().then((articles) => {
+      setAllArticles(articles);
+      setLoading(false)
+    });
+  }, []);
   return (
-    <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
-  )
+    <body>
```

Comment:
Using `body` here is resulting in an error because the `body` element can't be contained within a `div` and the whole react app is a `div` with the class `root` (you can inspect the page to confirm this)